local-cli/bundle/buildBundle.js

```diff
@@ -26,12 +28,17 @@ function buildBundle(args, config, output = outputBundle, packagerInstance) {
     // have other choice than defining it as an env variable here.
     process.env.NODE_ENV = args.dev ? 'development' : 'production';
 
+    let transformerPath = args.transformer;
+    if (!isAbsolutePath(transformerPath)) {
```

Comment:
Don't need the isAbsolutePath check. It's probably more consistent always to call path.resolve so that we always normalize the path.